docs/get-started/dev-containers.md

```diff
@@ -76,3 +76,305 @@ The preceding walkthrough demonstrates the streamlined process of creating a Dev
+  },
+  "onCreateCommand": "dotnet new install Aspire.ProjectTemplates::9.3.1 --force",
+  "postCreateCommand": "dotnet dev-certs https --trust && dapr init",
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "ms-dotnettools.csdevkit",
+        "ms-azuretools.vscode-dapr"
+      ]
+    }
+  }
+}
+```
+
+#### Dapr with external backends
+
+For more complex Dapr scenarios that use external backends (Redis, PostgreSQL), you can use Docker Compose:
+
+```json
+{
+  "name": ".NET Aspire with Dapr and Backends",
+  "image": "mcr.microsoft.com/devcontainers/dotnet:9.0-bookworm",
+  "features": {
+    "ghcr.io/devcontainers/features/docker-in-docker:2": {
+      "enableNonRootDocker": true
+    },
+    "ghcr.io/dapr/cli/dapr-cli:0": {}
+  },
+  "runArgs": [
+    "--sysctl",
+    "net.ipv6.conf.all.disable_ipv6=0"
+  ],
+  "onCreateCommand": "dotnet new install Aspire.ProjectTemplates::9.3.1 --force",
+  "postCreateCommand": [
+    "dotnet dev-certs https --trust",
+    "docker compose up -d",
+    "dapr init"
+  ],
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "ms-dotnettools.csdevkit",
+        "ms-azuretools.vscode-dapr",
+        "ms-azuretools.vscode-docker"
+      ]
+    }
+  }
+}
+```
+
+### Infrastructure and Bicep scenarios
+
+For .NET Aspire projects that include Bicep infrastructure files and Azure deployment:
+
+```json
+{
+  "name": ".NET Aspire with Azure Infrastructure",
+  "image": "mcr.microsoft.com/devcontainers/dotnet:9.0-bookworm",
+  "features": {
+    "ghcr.io/devcontainers/features/docker-in-docker:2": {
+      "enableNonRootDocker": true
+    },
+    "ghcr.io/azure/azure-dev/azd:0": {
+      "version": "latest"
+    }
+  },
+  "onCreateCommand": "dotnet new install Aspire.ProjectTemplates::9.3.1 --force",
+  "postStartCommand": "dotnet dev-certs https --trust",
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "ms-dotnettools.csdevkit",
+        "ms-azuretools.vscode-bicep",
+        "ms-azuretools.vscode-docker"
+      ]
+    }
+  }
+}
+```
+
+### Kubernetes deployment with Aspirate
+
+For scenarios where you want to deploy to Kubernetes using Aspirate:
+
+```json
+{
+  "name": ".NET Aspire with Aspirate",
+  "image": "mcr.microsoft.com/devcontainers/dotnet:9.0-bookworm",
+  "features": {
+    "ghcr.io/devcontainers/features/docker-in-docker:2": {
+      "enableNonRootDocker": true,
+      "moby": true
+    },
+    "ghcr.io/prom3theu5/aspirational-manifests/aspirate:latest": {}
+  },
+  "runArgs": [
+    "--sysctl",
+    "net.ipv6.conf.all.disable_ipv6=0"
+  ],
+  "onCreateCommand": "dotnet new install Aspire.ProjectTemplates::9.3.1 --force",
+  "postCreateCommand": [
+    "dotnet dev-certs https --trust",
+    "dotnet tool install -g aspirate --prerelease"
+  ],
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "ms-dotnettools.csdevkit",
+        "ms-azuretools.vscode-docker",
+        "redhat.vscode-yaml"
+      ]
+    }
+  }
+}
+```
```

Comment:
Let's remove this section for now please. If we're talking deployment at all, I'm not sure we should do so from the context of a Dev Container.